src/common/TALENTS/priest.ts

```diff
@@ -142,6 +142,13 @@ const talents = createTalentList({
     maxRanks: 2,
     entryIds: [103738],
   },
+  BURNING_VEHEMENCE_TALENT_DAMAGE: {
```

Comment:
This should go in `src/common/SPELLS/priest.ts` instead of here so that it doesn't get lost the next time we do a talent regeneration.